package.json

```diff
@@ -4,7 +4,7 @@
   "description": "◦ The Uniswap default token list",
   "main": "build/uniswap-default.tokenlist.json",
   "scripts": {
-    "test": "mocha",
+    "test": "mocha --timeout 10000",
```

Comment:
It might make sense to manually provide the timeout where it's necessary (using `this.timeout(n)`) instead of setting it globally and potentially slowing down the whole suite.